autopilot-daemon/pkg/utils/functions.go

```diff
@@ -145,3 +147,32 @@ func CreateJob(healthcheck string) {
 		klog.Info("Couldn't create Job ", err.Error())
 	}
 }
+
+func Initmodels() {
```

Comment:
I think this functionality would be better located in the existing `Initmetrics()` method. Do you agree? CC @cmisale

Reply:
Hm no I think I disagree because that init metrics is just about initializing Prometheus..
But `Initmodels` can be moved to the prometheus package in `utils/prometheus.go`.